source/quickstart.rst

```diff
@@ -0,0 +1,477 @@
+  >>> rhf_h2o = scf.RHF(mol_h2o)
+  >>> e_h2o = rhf_h2o.kernel()
+
+Besides the final converged ground-state energy, the mean-field object will further store the accompanying MO coefficients, occupations, etc. To illustrate how open-shell, possibly spin-polarized calculations are performed, different Hartree-Fock simulations of the O\ :sub:`2` dimer - with its triplet ground state - are given as (cf. `scf/02-rohf_uhf.py <https://github.com/pyscf/pyscf/blob/master/examples/scf/02-rohf_uhf.py>`_):
+
+  >>> mol_o2 = gto.M(atom='O 0 0 0; O 0 0 1.2', spin=2) # (n+2 alpha, n beta) electrons
+  >>> uhf_o2 = scf.UHF(mol_o2)
+  >>> uhf_o2.kernel()
+  >>> rohf_o2 = scf.ROHF(mol_o2)
+  >>> rohf_o2.kernel()
+
+Finally, a second-order, Newton-Raphson SCF method has been implemented, which is also applicable with most XC functionals, cf. :ref:`below <KSDFT>`. This algorithm needs orthonormal orbitals and their corresponding occupancies as initial guesses. These are automatically generated if none are provided (cf. `scf/22-newton.py <https://github.com/pyscf/pyscf/blob/master/examples/scf/22-newton.py>`_):
+
+  >>> rhf_h2o = rhf_h2o.newton()
+  >>> e_h2o = rhf_h2o.kernel()
+
+.. _KSDFT:
+
+Kohn-Sham Density Functional Theory
+-----------------------------------
+
+Running a KS-DFT calculation is every bit as straightforward as what we discussed :ref:`above <HF>` for an HF counterpart (cf. `dft/00-simple_dft.py <https://github.com/pyscf/pyscf/blob/master/examples/dft/00-simple_dft.py>`_):
+
+  >>> from pyscf import dft
+  >>> rks_h2o = dft.RKS(mol) # likewise for UKS and ROKS
+  >>> rks_h2o.xc = 'b3lyp'
+  
+Besides the use of predefined XC functionals (cf. `pyscf/dft/libxc.py <https://github.com/pyscf/pyscf/blob/master/pyscf/dft/libxc.py>`_ and `pyscf/dft/xcfun.py <https://github.com/pyscf/pyscf/blob/master/pyscf/dft/xcfun.py>`_ for the complete lists of
+available functionals), these can also be fully defined by the user (`dft/24-custom_xc_functional.py <https://github.com/pyscf/pyscf/blob/master/examples/dft/24-custom_xc_functional.py>`_), as can the angular and radial grids to be used in the calculation (`dft/11-grid_scheme.py <https://github.com/pyscf/pyscf/blob/master/examples/dft/11-grid_scheme.py>`_):
+
+  >>> rks_h2o.xc = '.2 * HF + .08 * LDA + .72 * B88, .81 * LYP + .19 * VWN' # B3LYP
+  >>> rks_h2o.grids.atom_grid = (100, 770)
+  >>> rks_h2o.grids.prune = None
+  >>> e_rks = rks_h2o.kernel()
+  
+The use of a combination of dense and sparse grids are particularly important whenever XC functionals with non-local dispersion (van der Waals) corrections are employed (cf. `dft/33-nlc_functionals.py <https://github.com/pyscf/pyscf/blob/master/examples/dft/33-nlc_functionals.py>`_):
+
+  >>> rks_c2 = dft.RKS(mol_c2)
+  >>> rks_c2.xc = 'wb97m_v'
+  >>> rks_c2.nlc = 'vv10'
+  >>> rks_c2.grids.atom_grid = (99,590)
+  >>> rks_c2.grids.prune = None
+  >>> rks_c2.nlcgrids.atom_grid = (50,194)
+  >>> rks_c2.nlcgrids.prune = dft.gen_grid.sg1_prune
```

Comment:
This should go into the DFT part of the user guide, which is still unclaimed

Reply:
Same comment as above, but I'd be happy to _help_ write the KS-DFT part of the user guide if need be.

Reply:
@januseriksen ok, I'll tag it as your responsibility ;)